### PR TITLE
telco5g: Prepare telco jobs for 4.14

### PR DESCRIPTION
**Pull request by sshnaidm**

### CODE REVIEW

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-telco5g-cnftests

**Comment by sshnaidm** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-telco5g-cnftests

File: ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh [hunk 1]
```diff
@@ -159,7 +161,7 @@ fi
 if [[ "$CNF_BRANCH" == *"4.12"* ]]; then
     create_tests_temp_skip_list_12
 fi
-if [[ "$CNF_BRANCH" == *"4.13"* ]] || [[ "$CNF_BRANCH" == *"master"* ]]; then
+if [[ "$CNF_BRANCH" == *"4.13"* ]] || [[ "$CNF_BRANCH" == *"4.14"* ]] || [[ "$CNF_BRANCH" == *"master"* ]]; then
```
Comment (yuvalk) [inline ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh:164]: will have to be it's own `if` when we branch out
but ok for now I guess

**Pull request merged**